### ISSUE TITLE
Invalid axis param value in graphmodel

### ISSUE THREAD

**Issue by Mrpatekful** [tensorflow/tfjs#1957]
#### TensorFlow.js version

"format": "graph-model", "generatedBy": "2.0.0-rc0", "convertedBy": "TensorFlow.js Converter v1.2.6"

#### Browser version

Google chrome 74.0.3729.108-1

#### Describe the problem or feature request

I have trained a simple embedding model in Python, but when I try to use it in tfjs, it raises an error.

Error: All values in axis param must be in range [-2, 2) but got axis 1004583828

#### Code to reproduce the bug / link to feature request

**The python code for the embedding model:**

```python
class MultiheadClassifier(Model):

    def __init__(self, vocab_size):
        super(MultiheadClassifier, self).__init__()

        self.transformer = Embedding(
            vocab_size, FLAGS.hidden_size)

        self.out_layers = [
            Dense(units=2),
            Dense(units=2),
            Dense(units=2),
            Dense(units=4),
            Dense(units=4)
        ]

    @tf.function(input_signature=(tf.TensorSpec(
        shape=(None, None), dtype=tf.int64), ))
    def call(self, inputs):
        outputs = self.transformer(inputs)

        logits = [
            layer(outputs) for 
            layer in self.out_layers
        ]

        return logits
```

**The tfjs code for evaluation:**

```javascript
const MODEL_URL = 'https://model-server.herokuapp.com/models/dcorr/model.json'
const model = await tf.loadGraphModel(MODEL_URL);
const inputs = tf.tensor([[1, 1]]);
const casted = tf.cast(inputs, 'int32');
model.predict(casted); // Error: All values in axis param must be in range [-2, 2) but got axis 1004583828
```

**Comment by rthadur** [comment]
@dsmilkov tried to reproduce the error but getting a different error when i tried in the [website](https://js.tensorflow.org/api/latest/#loadGraphModel) with above converted model , can you please assist.

**Comment by Mrpatekful** [comment]
Strangely the error has disappeared. 

```javascript
const modelUrl =
  'https://model-server.herokuapp.com/models/dcorr/model.json';
const model = await tf.loadGraphModel(modelUrl);
const inputs = tf.tensor([[1, 1]]);
const casted = tf.cast(inputs, 'int32');
const result = model.predict(casted);
console.log(result);
```
Output:
```text
Tensor
    [[[3.0487518, -3.0432105],
      [3.2661409, -2.2721605]]],Tensor
    [[[1.5872703, -2.5199966],
      [2.4837859, -3.7359493]]],Tensor
    [[[0.9032148, -2.4179547, 0.6162631, -0.7338544],
      [1.1808958, -1.6455753, 0.1840585, -0.1801595]]],Tensor
    [[[3.3952603, -2.4640663],
      [3.0766368, -1.8808024]]],Tensor
    [[[2.842907 , -2.7685816, 1.0244383, 0.6520141],
      [2.5355124, -2.6940522, 0.8816844, 0.5971178]]]
```

I don't realy now the cause of the issue, but it is now working well.
